README.md

```diff
@@ -1,28 +1,25 @@
-# `kcp` is a minimal Kubernetes API server
+# `kcp` is a control plane for applications running across multiple Kubernetes clusters
```

Comment:
```suggestion
# `kcp` is a highly multi-tenant control plane for applications with attachable Kubernetes clusters as compute
```

Reply:
Hrm, I liked some of your earlier wording better.

> kcp is a multi-tenant control plane for applications distributed across Kubernetes clusters and clouds

maybe

Reply:
Went with something slightly different